index.html

```diff
@@ -1,13 +1,201 @@
+            </p>
+          </div>
+          <div class="program">
+            <img class="program-icon" src="./img/icons/chat.png" alt="Forum" />
+            <h3 class="program-title">Forum</h3>
+            <p class="program-descr">
+              Listen to the speakers from various countries about the messages
+              of sharing and opening.
+            </p>
+          </div>
+          <div class="program">
+            <img
+              class="program-icon"
+              src="./img/icons/workshop.png"
+              alt="workshops"
+            />
+            <h3 class="program-title">Workshop</h3>
+            <p class="program-descr">
+              Listen to the speakers from various countries about the messages
+              of sharing and opening.
+            </p>
+          </div>
+          <div class="program">
+            <img
+              class="program-icon"
+              src="./img/icons/global-network.png"
+              alt="ignite"
+            />
+            <h3 class="program-title">Ignite</h3>
+            <p class="program-descr">
+              Listen to the speakers from various countries about the messages
+              of sharing and opening.
+            </p>
+          </div>
+        </div>
+        <a class="program-link" href="/">SEE THE HOLE PROGRAM</a>
+        <button class="join">Join FACEBOOK IG Conference</button>
+      </section>
+      <section class="featured-speaker">
+        <h2>featured-speaker</h2>
+        <hr />
+        <div class="speakers"></div>
+        <button class="more">
+          MORE
+          <i class="fas fa-sort-down fa-lg"></i>
+        </button>
+      </section>
+    </main>
+    <footer>
+      <section class="partner" id="partner">
+        <h2>Partner</h2>
+        <hr />
+        <div class="logos">
+          <img
+            class="partner-logo"
+            src="./img/logos/logo-airbnb-404px-grey-1.png"
+            alt="Airbnb logo"
+          />
+          <img
+            class="partner-logo"
+            src="./img/logos/unnamed.png"
+            alt="Google logo"
+          />
+          <img
+            class="partner-logo"
+            src="./img/logos/mozilla-logo-gray.png"
+            alt="mozilla logo"
+          />
+          <img
+            class="partner-logo"
+            src="./img/logos/logo-instantgames.png"
+            alt="Instant games logo"
+          />
+        </div>
+      </section>
+      <section class="footer-s">
+        <div class="logo-footer">
+          <div class="logopro">
+            <i class="fas fa-adjust fa-2x"></i>
+            <p>FACEBOOK </p>
+            <p>IG</p>
+          </div>
+        </div>
+        <div class="copyright">
+          2021 Facebook Instant Games Summit. <br />Some right Reserved.,<br />
+          <p class="chinese-lng">
+            吾伊杰杰, 诶艾哦伊艾 诶艾诶杰儿艾艾艾弗 吾吉艾西吉 艾娜 吾吉诶哦
+            开诶哦哦伊艾娜 哦勒 艾艾哦伊艾艾伊哦 屁娜伊
+          </p>
+        </div>
+      </section>
+    </footer>
```

Comment:
Following the guidelines these changes are needed in your footer: 
It contains a logo aligned to the left and text aligned to the right.

**Mobile Size**
- It has a white background.

**Desktop Size**
- It has a dark background.

Reply:
Thank you for your review,
I'm following this template design provided by Microverse to design my footer :

- for the desktop size :

![](https://i.ibb.co/MSjYgMN/Capture-Desktop.jpg)

- for the mobile size :

![](https://i.ibb.co/hRDZqgs/Capture-Mobile.jpg)

I couldn't get the required change.